### ISSUE TITLE
[DEPR]: Remove JWT_AUTH_REFRESH_COOKIE

### ISSUE THREAD

**Issue by robrap** [openedx/public-engineering#190]
### Proposal Date

12 April 2023

### Target Ticket Acceptance Date

26 April 2023

### Earliest Open edX Named Release Without This Functionality

Quince - 2023-10

### Rationale

The setting ``JWT_AUTH_REFRESH_COOKIE`` is meaningless and unused, and should be cleaned up to avoid confusion.

In the very early days of introducing MFEs, we _thought_ we were going to need this cookie in addition to the JWT cookie. However, it turned out we didn't need it, but the setting stuck around the contagion of it (being in cookiecutter and other template libraries) has resulted in it uselessly being copied to many repos.

### Removal

The setting ``JWT_AUTH_REFRESH_COOKIE`` can simply be removed with no ramifications.

### Replacement

No replacement needed.

### Additional Info

See https://github.com/search?q=%28org%3Aedx+OR+org%3Aopenedx%29+JWT_AUTH_REFRESH_COOKIE&type=code&p=5

Note: edx.org has private uses that should also be cleaned out to avoid confusion, which are in the above search for those who have access.

**Comment by Yagnesh1998** [comment]
@dianakhuang  https://github.com/openedx/edx-platform/pull/32664 This pull request all tests passed successfully

**Comment by robrap** [comment]
I moved the edx org work to a separate ticket that would not block this DEPR: https://github.com/edx/edx-arch-experiments/issues/353.

**Comment by robrap** [comment]
@Yagnesh1998: If you could start by removing this from openedx/configuration. See https://github.com/search?q=repo%3Aopenedx%2Fconfiguration+JWT_AUTH_REFRESH_COOKIE&type=code. Once this is done, you can fix other repos. Thank you.

**Comment by Yagnesh1998** [comment]
assign me

**Comment by lokidil** [comment]
anyone is working on this ? is this closed ?

**Comment by Yagnesh1998** [comment]
@lokidil I'm working on it, and several GitHub repos still need to be changed. Thanks.
